InnerEye/ML/lightning_container.py

```diff
@@ -288,6 +301,35 @@ def create_lightning_module_and_store(self) -> None:
             self._model._optimizer_params = create_from_matching_params(self, OptimizerParams)
             self._model._trainer_params = create_from_matching_params(self, TrainerParams)
 
+    def get_cross_validation_hyperdrive_config(self, run_config: ScriptRunConfig) -> HyperDriveConfig:
+        """
+        Returns a configuration for AzureML Hyperdrive that varies the cross validation split index.
+        :param run_config: The AzureML run configuration object that training for an individual model.
+        :return: A hyperdrive configuration object.
+        """
+        return HyperDriveConfig(
+            run_config=run_config,
+            hyperparameter_sampling=GridParameterSampling(
+                parameter_space={
+                    CROSS_VALIDATION_SPLIT_INDEX_TAG_KEY: choice(list(range(self.number_of_cross_validation_splits)))
+                }),
+            primary_metric_name=TrackedMetrics.Val_Loss.value,
```

Comment:
Is it up to the user to make sure they log a metric with the same name in their model configs?

Reply:
That's interesting. I'm not sure, @ant0nsc?

Reply:
Done. I've added a comment in the method you highlighted and some directives in the BYOM documentation. Both point to the example in `HelloRegression(LightningModule)` where we provide the val/Loss logging.